### PR TITLE
Change main API to include major_info

### PR DESCRIPTION
**Pull request by sonowz**
Also removed static file serving in backend server, since it's no longer needed.

### CODE REVIEW

File: backend/api/login/tests.py [hunk 1]
```diff
@@ -33,11 +39,54 @@ def test_file_request(self):
...
-        response = self.req_class.get_rule(request)
-        self.assertEqual(response, 'sample_cse_2016')
+        major_info, rule = self.req_class.get_rules(request)[0]
+        self.assertEqual(rule, 'sample_cse_2016')
+
+
+class TestTreeToTable(TestCase):
+    def setUp(self):
+        self.sample_rule = 'sample_cse_2016'
+        sugang_list_dir = os.path.join(settings.BASE_DIR, 'test_sample/sugang_list.pickle')
```
Comment (skystar-p) [inline backend/api/login/tests.py]: `os.path.join` can take multiple arguments. So `os.path.join(settings.BASE_DIR, 'test_sample', 'sugang_list.pickle')` is right.

**Review by skystar-p — CHANGES_REQUESTED** [review]
review review

**Comment by skystar-p** [comment]
Can you show us what data is saved in `tables`?

**Comment by sonowz** [comment]
It is `dict` which will be later converted to `JSON` in #30 main data response. The format is same.

**Comment by sonowz** [comment]
Also fixed bug in `login/view.py`
```python
request.data['major_info'] is immutable
```

**Review by skystar-p — APPROVED** [review]

**Pull request merged**